apps/wallet-mobile/src/yoroi-wallets/cardano/api/utils.ts

```diff
@@ -125,3 +125,17 @@ export const toTokenFingerprint = ({
   const assetFingerprint = AssetFingerprint.fromParts(Buffer.from(policyId, 'hex'), Buffer.from(assetNameHex, 'hex'))
   return assetFingerprint.fingerprint()
 }
+
+export const transformCIP0068AssetNameHex = (assetNameHex) => {
+  const refprefix = '000643b0'
+  const usrprefix = '000de140'
+  const stringsToRemove = [refprefix, usrprefix]
+
+  let transformedAssetNameHex = assetNameHex
+
+  stringsToRemove.forEach((str) => {
+    transformedAssetNameHex = transformedAssetNameHex.replace(str, '')
```

Comment:
Not sure I understood completely the issue with CIP68.
It sounds like asset names come with a prefix to indicate something?
With this approach, it'll remove both strings, regardless of whether they are the prefix or not.

I would recommend a replace with regex, and prefix it with ` ^` so that it only matches at the beginning.

Reply:
finally a yoroi-lib tool has been used